File: hotel_list/v1/get/hotels_by_geocode/Python SDK/hotel_list.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import ResponseError, Client

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    Get list of hotels by a geocode
    '''
    response = waconvo.reference_data.locations.hotels.by_geocode.get(longitude=2.160873,latitude=41.397158)

    print(response.data)
except ResponseError as error:
    raise error

File: hotel_list/v1/get/hotels_by_geocode/Node SDK/hotel_list.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // List of hotels in Paris
    const response = await waconvo.referenceData.locations.hotels.byGeocode.get(
      {
        latitude: 48.83152,
        longitude: 2.24691,
      }
    );

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();